packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.styles.ts

```diff
@@ -10,6 +11,9 @@ const GlobalClassNames = {
 export const getStyles = (props: IDetailsRowCheckStyleProps): IDetailsRowCheckStyles => {
   const { theme, className, isHeader, selected, anySelected, canSelect, compact, isVisible } = props;
   const classNames = getGlobalClassNames(GlobalClassNames, theme);
+  const { rowHeight, compactRowHeight } = DEFAULT_ROW_HEIGHTS;
+
+  const height = isHeader ? 32 : compact ? compactRowHeight : rowHeight;
```

Comment:
Could you combine this and do 

`const height = isHeader || compact ? compactRowHeight : rowHeight`

Reply:
@natalieethell I intentionally left that 32 value in there as not to create a confusion in the future thinking that the header's height is deriving from the `compactRowHeight`. Even though the values are the same, their source should be different. To consolidate it further I will instead export that value from the `DetailsHeader.styles.ts` to keep it in one source.

As for the `Check` alignment in RTL, I will later follow up with a separate PR in trying to replace the glyphs used to construct the Check with different ones, which should not have this issue. I already talked to Ben about it and we have the glyphs available to do that. I only need to find a way of replacing them without a breaking change. Meanwhile, I will make this temporary fix you suggested. Thanks for noticing it. 👍

Reply:
Sounds good!